src/components/ColorPicker/ColorPicker.scss

```diff
@@ -0,0 +1,364 @@
+@import "../../constants/style";
+
+.color-picker {
+  z-index: 220;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  padding-bottom: $spacing--xxs;
+  box-shadow: 0 0 20px rgba(55, 62, 79, 0.12);
+  background-color: $gray--300;
+  list-style: none;
+  padding-left: 0;
+  margin-top: auto;
+  border-radius: $rounded--full;
+}
+
+.color-picker > li > button {
+  cursor: pointer;
+  border: none;
+  background: none;
+  padding: 0;
+  width: 40px;
+  height: 40px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  &:focus-within {
+    > .column__header-color-option {
+      &--planning-pink {
+        border: 3px solid $pink--100;
+      }
+      &--backlog-blue {
+        border: 3px solid $blue--100;
+      }
+      &--poker-purple {
+        border: 3px solid $purple--100;
+      }
+      &--value-violet {
+        border: 3px solid $violet--100;
+      }
+      &--goal-green {
+        border: 3px solid $green--100;
+      }
+      &--yielding-yellow {
+        border: 3px solid $yellow--100;
+      }
+      &--online-orange {
+        border: 3px solid $orange--100;
+      }
+    }
```

Comment:
by adding the color class name to this or a parent element, you can replace all of this with `border: 3px: var(--accent-color--100)`. It will automatically pick the correct one :)

Reply:
of course, these also exist, for `--200`, `--300`, and so on up to `--800`. In addition to that, `--accent-color--xxx-rgb` also exists, like `--accent-color--100-rgb`, if you ever need to adjust things like the alpha value.

Oh and also there are the shortcuts `--accent-color--light` and `--accent-color--dark`, which equate to `--500` and `--400` (also available as `-rgb`) ✌️

Reply:
didn't know that was a thing, thanks!